src/ng2-facebook-sdk.ts

```diff
@@ -138,11 +138,12 @@ export class FacebookService {
      * This synchronous function returns back the current authResponse.
      * @returns {FacebookAuthResponse}
      */
-    getAuthResponse(): FacebookAuthResponse {
+    getAuthResponse(): FacebookAuthResponse | null {
```

Comment:
Thanks for the changes. All look perfect.

This `| null` is not needed. I think even `return null;` below is not needed.